dashbuilder/dashbuilder-client/dashbuilder-displayer-client/src/main/java/org/dashbuilder/displayer/client/RendererManager.java

```diff
@@ -56,15 +62,28 @@ public RendererManager(SyncBeanManager beanManager) {
 
     @PostConstruct
     private void init() {
+        rendererSettings.call((String defaultUUIDOp) -> lookupRenderers(defaultUUIDOp))
+                        .userDefaultRenderer();
+    }
+
+    private void lookupRenderers(String defaultUUID) {
         Collection<SyncBeanDef<RendererLibrary>> beanDefs = beanManager.lookupBeans(RendererLibrary.class);
+        if (defaultUUID != null && ! defaultUUID.isEmpty()) {
+            beanDefs.stream()
+                    .map(SyncBeanDef::getInstance)
+                    .filter(render -> render.getUUID().equals(defaultUUID))
```

Comment:
I'm just wondering what will happen if NONE of the renderers will have uuid matching the user-selected
E.g what happens when user sets `-Dorg.dashbuilder.renderer.default=nonexistentUUID`?
Maybe that scenario is ok, but I'm not sure just from looking at the code..

Reply:
@jhrcek  updated with a new commit, would you please take a look? If user set a bad value for the system property it is simply ignored. I added tests for the changes as well.

Reply:
@jhrcek Regarding the issue you found, I could reproduce it and found a pattern, it only happens when there's charts with "no data". I will reproduce it on dashbuilder and let you know.

Reply:
Just for context, @jhrcek  tested this PR and saw the following error --> http://i.imgur.com/7BeOyKD.png

I fixed AF-1749 and the error seems to be gone. In fact, the error Jan saw is a generic error, which hides the root cause, and the root cause seems to be indeed AF-1749.

AF-1749 was fixed on the PR [1]. @jhrcek  Please let me know if you prefer to check  [1] and then come back to this PR or I can cherry pick the commit from [1] to this PR and you can test everything at once.

Thanks!

[1] https://github.com/kiegroup/appformer/pull/584